mubench.pipeline/tests/data/test_detector_run.py

```diff
@@ -0,0 +1,195 @@
+import logging
+from os.path import join
+from tempfile import mkdtemp
+from unittest.mock import MagicMock, patch, ANY
+
+from nose.tools import assert_equals
+
+from data.detector_run import DetectorRun, Result
+from data.finding import Finding
+from tasks.configurations.detector_interface_configuration import key_findings_file, key_run_file
+from tests.data.stub_detector import StubDetector
+from tests.test_utils.data_util import create_version, create_project
+from utils.io import remove_tree
+from utils.shell import CommandFailedError
+
```

Comment:
All three test classes in the file test the `DetectorRun`. Please either merge them into one class or give each class a name that reveals which aspect of the run is tested, e.g., `TestDetectorRunLoadsState`, `TestDetectorRunSavesState`, `TestDetectorRunLoadsFindings`.